rag/flow/tests/test_token_chunker.py

```
@@ -579,3 +622,26 @@ def test_one_mode_text_strips_coord_tags():
+        prev = [
+            ("aaaaa", [[1, 0, 10, 0, 5]]),
+            ("bbbbb", [[2, 0, 20, 0, 8]]),
+        ]
+        # total = 10 visible runes; overlap tail is [8, 10).
+        tail = fn(prev, 8)
+        assert len(tail) == 1, tail
+        text, pos = tail[0]
+        assert text == "bb", text
+        assert pos == [[2, 0, 20, 0, 8]], pos
+        # head item (page 1) must not leak into the overlap tail.
+        for t, p in tail:
```

Ruff (0.16.2)

[warning] Loop control variable `t` not used within loop body

Rename unused `t` to `_t`

(B007)